filebeat/harvester/reader/multiline.go

```diff
@@ -241,9 +261,11 @@ func (mlr *Multiline) addLine(m Message) {
 	}
 
 	space := mlr.maxBytes - sz
-	spaceLeft := (mlr.maxBytes <= 0 || space > 0) &&
-		(mlr.maxLines <= 0 || mlr.numLines < mlr.maxLines)
-	if spaceLeft {
+
+	bytesLeft := (mlr.maxBytes <= 0 || space > 0)
+	linesLeft := (mlr.maxLines <= 0 || mlr.numLines < mlr.maxLines)
```

Comment:
maybe use names indicating we checking something. `XLeft` sounds like numeric value (got a little confused by if condition).